tutorials/text/text_intro.py

```diff
@@ -93,3 +96,337 @@
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+x1 = np.linspace(0.0, 5.0, 100)
+y1 = np.cos(2 * np.pi * x1) * np.exp(-x1)
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(bottom=0.15, left=0.2)
+ax.plot(x1, y1)
+ax.set_xlabel('time [s]')
+ax.set_ylabel('Damped oscillation [V]')
+
+plt.show()
+
+###############################################################################
+# The x- and y-labels are automatically placed so that they clear the x- and
+# y-ticklabels.  Compare the plot below with that above, and note the y-label
+# is to the left of the one above.
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(bottom=0.15, left=0.2)
+ax.plot(x1, y1*10000)
+ax.set_xlabel('time [s]')
+ax.set_ylabel('Damped oscillation [V]')
+
+plt.show()
+
+###############################################################################
+# If you want to move the labels, you can specify the *labelpad* kyeword
+# argument, where the value is points (1/72", the same unit used to specify
+# fontsizes).
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(bottom=0.15, left=0.2)
+ax.plot(x1, y1*10000)
+ax.set_xlabel('time [s]')
+ax.set_ylabel('Damped oscillation [V]', labelpad=18)
+
+plt.show()
+
+###############################################################################
+# Or, the labels accept all the `~matplotlib.text.Text` keyword arguments,
+# including *position*, via which we can manually specify the label positions.
+# Here we put the xlabel to the far left of the axis.  Note, that the
+# y-coordinate of this position has no effect - to adjust the y-position we
+# need to use the *labelpad* kwarg.
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(bottom=0.15, left=0.2)
+ax.plot(x1, y1)
+ax.set_xlabel('time [s]', position=(0., 1e6),
+                      horizontalalignment='left')
+ax.set_ylabel('Damped oscillation [V]')
+
+plt.show()
+
+##############################################################################
+# All the labelling in this tutorial can be changed by manipulating the
+# `matplotlib.font_manager.FontProperties` method, or by named kwargs to
+# `~matplotlib.axes.Axes.set_xlabel`
+
+from matplotlib.font_manager import FontProperties
+
+font = FontProperties()
+font.set_family('serif')
+font.set_name('Times New Roman')
+font.set_style('italic')
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(bottom=0.15, left=0.2)
+ax.plot(x1, y1)
+ax.set_xlabel('time [s]', fontsize='large', fontweight='bold')
+ax.set_ylabel('Damped oscillation [V]', fontproperties=font)
+
+plt.show()
+
+##############################################################################
+# Finally, we can use native TeX rendering in all text objects and have
+# multiple lines:
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(bottom=0.2, left=0.2)
+ax.plot(x1, np.cumsum(y1**2))
+ax.set_xlabel('time [s] \n This was a long experiment')
+ax.set_ylabel('$\int\ Y^2\ dt\ \ [V^2 s]$')
```

Comment:
needs to be a raw string (otherwise the backslash escapes are invalid)

Reply:
? They render fine.

Reply:
https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals
> Changed in version 3.6: Unrecognized escape sequences produce a DeprecationWarning. In some future version of Python they will be a SyntaxError.

(run with python -Wd to see the deprecation warnings)